spec/functional/provider/whyrun_safe_ruby_block_spec.rb

```diff
@@ -43,7 +43,7 @@
     end
 
     it "updates the evil laugh, even in why-run mode" do
-      new_resource.run_action(new_resource.action)
+      Array(new_resource.action).each {|action| new_resource.run_action(action) }
```

Comment:
As before, there are a lot of ways this can still be a non-array when interacting with legacy code, forcing the up-cast is safest.